website/src/views/pageStateHandlers/PageStateHandler.ts

```diff
@@ -66,10 +68,37 @@ export function toLapisFilterWithoutVariant(
         {} as { [key: string]: string | undefined },
     );
 
+    const numberRangeFilters = Object.entries(pageState.datasetFilter.numberFilters).reduce(
+        (acc, [lapisField, numberRange]) => {
+            if (numberRange === undefined) {
+                return acc;
+            }
+
+            const fromValue = numberRange.min
+                ? {
+                      [`${lapisField}From`]: String(numberRange.min),
+                  }
+                : {};
+            const toValue = numberRange.max
+                ? {
+                      [`${lapisField}To`]: String(numberRange.max),
+                  }
+                : {};
+
+            return {
+                ...acc,
+                ...fromValue,
+                ...toValue,
+            };
```

Comment:
```suggestion
            return {
                ...acc,
                [`${lapisField}From`]: numberRange.min === undefined ? undefined : String(numberRange.min),
                [`${lapisField}To`]: numberRange.max === undefined ? undefined : String(numberRange.max),
            };
```

Doesn't that already do the job?

(We could even get rid of the early return by using `numberRange?.min`)

Reply:
Yes and now. This way, we set a key "numberFrom" to undefined, which is then also send to lapis. I wanted to only send filters, that have a value. Therefore, I need the construct with the `{}`, which is then left out during destructuring. 

I added a test, to show this.

Reply:
It doesn't really matter, `undefined` is made to be "not present". When serializing an object to JSON, then all keys where the value is `undefined` will be skipped.

It would only matter if we overwrote keys. But I think it's safe to assume that in this context, we will never have duplicate keys on those objects. -> I'd prefer the shorter code which writes a couple of "useless" `undefined`s.

Reply:
As discussed. We leave it like this, after some reformatting.